rpm/aardvark-dns.spec

```diff
@@ -78,6 +96,14 @@ tar fx %{SOURCE1}
 %install
 %{__make} DESTDIR=%{buildroot} PREFIX=%{_prefix} install
 
+%{__install} -d -p %{buildroot}%{_datadir}/%{name}/test
+%{__cp} -rp test/* %{buildroot}%{_datadir}/%{name}/test/
+%{__rm} -rf %{buildroot}%{_datadir}/%{name}/test/tmt/    
+
+# Add empty check section to silence rpmlint warning.
+# No tests meant to be run here.
+%check
```

Comment:
Looking around online this seems to say


> Command or series of commands to test the software. This normally includes things such as unit tests.

https://rpm-packaging-guide.github.io/

So should we run unit tests there? I see you do have the unit test listed under the actual tests which I guess only run upstream? I have experience around that here so any reason not to run the unit test at least part of the normal package build?

Reply:
Do the unit tests depend on the generated aardvark-dns binary at all? If not, I don't see any point in running them in `%check` .

Reply:
Unit test do not depend on the build binary in any way. I don't mind either way we run them and using tmt is perfectly fine

Reply:
Looking at the rpm build log that does indeed run the unit tests as part of the rpm build so I guess if unit tests fail we do not get a rpm. Whenever that is preferable or not I cannot judge and leave it to you if you want that.

The unit tests here are not particular distro depended so I think it is  fine if we only run it upstream

Reply:
ack thanks. unit test failures will be visible to us upstream anyway, so I'd say let's not keep them as additional blockers for the rpm. And not blocking on unit tests for rpm has worked mostly fine for us so far AFAICT.

Removing it from %check.